src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, slug, sex } = person;
```

Comment:
Ensure that the `Person` type includes `name`, `slug`, and `sex` properties, as they are destructured from the `person` object and used in this component.